Tasks/IISWebAppDeployment/task.json

```diff
@@ -0,0 +1,416 @@
+{
+    "id": "89A3A82D-4B3E-4A09-8D40-A793849DC94F",
+    "name": "IISWebAppDeployment",
+    "friendlyName": "[Deprecated] IIS Web App Deployment",
+    "description": "Deploy by MSDeploy, create/update website & app pools",
+    "helpMarkDown": "[More Information](https://aka.ms/iiswebappdeployreadme)",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 23
```

Comment:
Please confirm if the patch version was updated.

Reply:
yes, it was bumped up